08-Placing-Barriers-in-the-Simulation-Box/05-Wall-potential-HPMC.ipynb

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
<p>Which documentation? For the specific shape? Or the specific wall? Perhaps an inclusion of the link could be helpful on the word "<a href="https://hoomd-blue.readthedocs.io/en/stable/module-hpmc-external-wall.html" target="_blank">documentation</a>".</p><p> </p><p>On the documentation it is written:</p><p>"See the individual subclasses of&nbsp;<code><a href="https://hoomd-blue.readthedocs.io/en/stable/module-hpmc-integrate.html#hoomd.hpmc.integrate.HPMCIntegrator" target="_blank">hoomd.hpmc.integrate.HPMCIntegrator</a></code>&nbsp;for their wall support." which I find slightly more clear.</p><p> </p><p>Also, this is the only use of the word "inactive" in the tutorial. Maybe find a consistent word to describe the side with and with out particles. "inaccessible" or "prohibited" may be more appropriate. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>